jflex-maven-plugin/pom.xml

```diff
@@ -44,7 +44,7 @@
     <url>https://github.com/jflex-de/jflex/issues</url>
   </issueManagement>
   <prerequisites>
-    <maven>3.0.1</maven>
+    <maven>${maven.version}</maven>
```

Comment:
We need a very recent version of Maven to compile JFlex due to a bug in maven-shade-plugin.

Are you sure we actually need the same to execute the plugin? It sounds to me that 3.0 should be enough.

Reply:
Maven version 3 should really be enough. The annotation were introduced  back then.